src/modules/corp-type-module/corp-type-module.ts

```diff
@@ -16,14 +16,14 @@ export enum CorpTypeCd {
   CCC_CONTINUE_IN = 'CCC',
   CEMETARY = 'CEM',
   COOP = 'CP',
-  BC_CORPORATION = 'CR', // SPECIAL NAMEREQUEST-ONLY ENTITY TYPE
+  CORPORATION = 'CR', // SPECIAL NAMEREQUEST-ONLY ENTITY TYPE
```

Comment:
Renamed to better match the various descriptions (and not confuse it with `BC`).